### ISSUE TITLE
Can't place construction ghosts on top of exposed floor rods

### ISSUE THREAD

**Issue by K-Dynamic** [space-wizards/space-station-14#33173]
## Description

You can't place construction ghosts on top of exposed floor rods due to an "obstruction." This is a problem for fixing breaches or building walls on the edges of grids (shuttles and station).

Notably you can build furniture or wall girders on valid tiles (hull or floor plating), unanchor them, then move and anchor them on top of exposed floor rods.

**Reproduction**

1. Find an exposed floor rod (either by spacing or placing rods into space)
2. Try to place construction ghost

**Screenshots**

***Unsuccessful construction ghost***
![image](https://github.com/user-attachments/assets/f8358285-1b11-4139-858c-1df46e206fab)

***Successful construction ghost***
![image](https://github.com/user-attachments/assets/86264e1a-2ad2-4807-bf7a-08d6325f0478)

**Comment by TurboTrackerss14** [comment]
Intentional by my recollection. 
Walls need to be on plating. 

The fact you can even anchor a girder onto lattice is the realer issue here. 

>Just make plating for the lattice chief..

**Comment by Anzarot121** [comment]
Yes, this is a serious problem. This is especially annoying when making radiators or lights

**Comment by VerinSenpai** [comment]
Add this to my weekend list.
